solutions/poi-18-stone.mdx

```diff
@@ -0,0 +1,84 @@
+---
+id: poi-18-stone
+source: POI 2018
+title: Pionek (stone)
+author: Albert Ye
+---
+
+## Solution
+This question can be solved through radial sweepline. Therefore, we sort our list of vectors by angle from the horizontal counterclockwise. This step comprises the bulk of the process, at $O(n \log n)$.
```

Comment:
```suggestion
This question can be solved through radial sweepline. Therefore, we sort our list of vectors by angle from the horizontal counterclockwise. This step comprises the bulk of the process, at $\mathcal{O}(n \log n)$.
```

Reply:
also probably could be worded better (horizontal counterclockwise). Maybe "in a counterclockwise fashion" would suffice.